gradle/libs.versions.toml

```diff
@@ -32,6 +32,7 @@ lint-api = { group = "com.android.tools.lint", name = "lint-api", version.ref =
 ktlint-cli = { group = "com.pinterest.ktlint", name = "ktlint-cli", version.ref = "ktlint" }
 ktlint-ruleset = { group = "com.pinterest.ktlint", name = "ktlint-cli-ruleset-core", version.ref = "ktlint" }
 material = "com.google.android.material:material:1.9.0"
+google-accompanist = "com.google.accompanist:accompanist-systemuicontroller:0.32.0"
```

Comment:
Google-accompanist is not needed to change the color of the status bar. Look at my answer on StackOverflow: https://stackoverflow.com/questions/73756866/jetpack-compose-cant-make-statusbar-completely-transparent/74593754#74593754. It could be achieved without 3rd party library.

Reply:
Done